assets/js/vendor/js.cookie.js

```diff
@@ -0,0 +1,163 @@
+/*!
+ * JavaScript Cookie v2.2.1
```

Comment:
I haven't but the library has a "No Conflict" mode we can run it in (and that this does run it in) which exists to for people like us:

https://github.com/gocodebox/lifterlms/pull/928/files/159d2d0109be3540e617118d1b220a3325a669bf#diff-dab5ab66ec9abcdd1fe9a6972fd55e1eR15

See the docs: https://github.com/js-cookie/js-cookie#namespace-conflicts

Of course... this assumes that anyone else using the library is also running it in noConflict mode otherwise we could end up with conflicts...

Reply:
Yeah, well I don't trust the noConflict thing :). 
I mean, yes it makes sense, but if you look at the bootstrap js it's full on noConflict yet we have conflicts with b3 and b4 collapse... Ok, in that case the problem is that there are reactions to dom events.
With the noConflict "you're fine" things do not break but you don't know which version you're using at runtime (right?!).